packages/editor/src/components/post-card-panel/style.scss

```
@@ -35,6 +35,10 @@
 
 .editor-post-card-panel__icon.is-sync {
 	fill: var(--wp-block-synced-color);
+
+	& + h2 {
+		color: var(--wp-block-synced-color);
+	}
```

```suggestion
	& + .editor-post-card-panel__title {
		color: var(--wp-block-synced-color);
	}
```

This is just my preference, so we may not need to change it.